### DISCUSSION TITLE
Paste within Java string

### DISCUSSION THREAD

**Discussion by Adam-001** [redhat-developer/vscode-java#3218] [Q&A]
![image](https://github.com/redhat-developer/vscode-java/assets/44825184/9d2a6529-6f78-4640-8bbc-30efa6c36d5f)
How to turn off this feature is really annoying

**Comment by fbricon** [comment]
To paste the normal way:
`Shift + Ctrl + V`

To disable it:
"editor.pasteAs.enabled": "false"
"editor.pasteAs.showPasteSelector": "never"

See https://code.visualstudio.com/updates/v1_79#_paste-as for more info
